packages/standard-components/manifest.json

```diff
@@ -111,6 +111,42 @@
         "type": "datasource",
         "label": "Data",
         "key": "datasource"
+      },
+      {
+        "type": "text",
+        "label": "No Rows Message",
+        "key": "noRowsMessage"
```

Comment:
We might want to add a `defaultValue` for this of "Feed me some data", because if someone edits this field and clears it, I think the empty string value will override the default prop in the component and mean there isn't any string at all. I'd probably remove the default prop in the component (so that in can be set to nothing if desired) and add a `defaultValue` here.